underfs/hdfs/src/test/java/alluxio/underfs/hdfs/HdfsUnderFileSystemTest.java

```diff
@@ -50,7 +52,30 @@ public void prepareConfiguration() throws Exception {
     org.apache.hadoop.conf.Configuration conf = new org.apache.hadoop.conf.Configuration();
     mMockHdfsUnderFileSystem.prepareConfiguration("", conf);
     Assert.assertEquals("org.apache.hadoop.hdfs.DistributedFileSystem", conf.get("fs.hdfs.impl"));
-    Assert.assertFalse(conf.getBoolean("fs.hdfs.impl.disable.cache", false));
+    Assert.assertTrue(conf.getBoolean("fs.hdfs.impl.disable.cache", false));
     Assert.assertNotNull(conf.get(PropertyKey.UNDERFS_HDFS_CONFIGURATION.toString()));
   }
+
+  /**
+   * Tests the HDFS client caching is disabled.
+   *
+   * @throws Exception
+   */
+  public void disableHdfsCacheTest() throws Exception {
+    // create a default hadoop configuration
+    org.apache.hadoop.conf.Configuration hadoopConf = new org.apache.hadoop.conf.Configuration();
+    String underfsAddress = "hdfs://localhost";
+    URI u = new URI(underfsAddress);
+    org.apache.hadoop.fs.FileSystem hadoopFs = org.apache.hadoop.fs.FileSystem.get(u, hadoopConf);
+    // use the replication to test the default value
+    Assert.assertEquals("3", hadoopFs.getConf().get("dfs.replication"));
+
+    // create a new configuration with updated dfs replication value
+    org.apache.hadoop.conf.Configuration hadoopConf1 = new org.apache.hadoop.conf.Configuration();
+    hadoopConf1.set("dfs.replication", "1");
```

Comment:
do we need to change the configuration for this test to work? I thought it wouldn't cache at all, even with the same configuration